pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -543,3 +562,34 @@ func (i *Gatherer) ClusterVersion() *configv1.ClusterVersion {
 	defer i.lock.Unlock()
 	return i.lastVersion
 }
+
+// ConfigMapAnonymizer implements serialization of configmap
+// and potentially anonymizes if it is a certificate
+type ConfigMapAnonymizer struct{ v []byte }
+
+// Marshal implements serialization of Node with anonymization
+func (a ConfigMapAnonymizer) Marshal(_ context.Context) ([]byte, error) {
+	return []byte(anonymizeConfigMap(a.v)), nil
+}
+
+func anonymizeConfigMap(dv []byte) string {
```

Comment:
What if there's another type of sensitive data in the config map? Like a user/password?

I'm asking because I'm not sure what could be inside `openshift-config`.

Reply:
I havent noticed anything except the certificates, which are being anonymized. If there is anything, I would need to handle it case by case, but so far I dont know about anything else.

Reply:
Not sure if someone else could help us here because if we end up collecting a password it will be too late to fix something.